### PR TITLE
Login

### PR DESCRIPTION
**Pull request by tomaskourim**

### CODE REVIEW

File: descriptive_statistics.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 import numpy as np
 import pandas as pd
 
-import constants
+import config
```
Comment (josefondrej) [inline descriptive_statistics.py]: In other files a different format is used, i.e. `from config import CONST`

File: live_betting/bookmaker.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+from selenium import webdriver
+
+from live_betting.config_betting import WEBDRIVER_PATH
+
+
+class Bookmaker:
+    def __init__(self, url):
+        self._home_url = url
```
Comment (josefondrej) [inline live_betting/bookmaker.py]: Consider renaming `url > home_url`

File: live_betting/utils.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+from selenium import webdriver
+
+
+def load_fb_credentials(credentials_file_path: str) -> Tuple[str, str]:
+    credentials = json.load(open(credentials_file_path, "rb"))
+    return credentials.get("username"), credentials.get("password")
+
+
+def write_xpath(driver: webdriver, xpath: str, text: str):
```
Comment (josefondrej) [inline live_betting/utils.py:12]: Wouldn't it make more sense to make a wrapper class for the webdriver and add these methods to it?

File: main.py [hunk 4]
```diff
@@ -9,7 +9,7 @@
 import scipy.optimize as opt
 import scipy.stats as stat
 
-import constants
+import config
```
Comment (josefondrej) [inline main.py:12]: Again inconsistent with the `from config import CONST` which is used in other files.

Comment (tomaskourim) [reply main.py:12]: Not clearly, this time.

File: live_betting/utils.py (hunk 3 above)
Comment (tomaskourim) [reply live_betting/utils.py:12]: It would be better to have a package for it. Wrapper is just some so-so solution as is utils file.

**Pull request merged**